components/avatar/Avatar.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * Author: bladimirardiles
+ * Component File Name: Avatar.js
+ * Component Name: Avatar
+ * Project: design-system
+ * Date: Sat 09/12/2023 - 11:34
+ */
+
+import React, { useId, useMemo } from 'react';
+import { IAvatarComponentProps } from './avatar.interface';
+import Button from '../button/Button';
+import { EIConTypes } from '../icon/icon.enum';
+import Image from '../image/Image';
+import { EImageLoadingType } from '../image/image.enum';
+import { get as _get } from 'lodash';
+import { File } from 'buffer';
+import { EAvatarSizes } from './avata.enum';
+import {
+  editableLabel,
+  topContainerLg,
+  topContainerMd,
+  topContainerSm,
+  topContainerXl,
+} from './avatar-css';
+
+const Avatar = ({
+  imgSrc,
+  onError,
+  onSuccess,
+  name,
+  editable,
+  size = EAvatarSizes.SM,
+}: IAvatarComponentProps) => {
+  // const cssClasses = modifierState(componentName, [], []);
+
+  const topContainerClasses = useMemo(() => {
+    if (size === EAvatarSizes.XL || editable) {
+      return topContainerXl.join(' ');
+    } else if (size === EAvatarSizes.LG) {
+      return topContainerLg.join(' ');
+    } else if (size === EAvatarSizes.MD) {
+      return topContainerMd.join(' ');
+    } else if (size === EAvatarSizes.SM) {
+      return topContainerSm.join(' ');
+    }
+  }, [size, editable]);
+
+  const identifier = useId();
+  return (
+    <div className="c-avatar relative">
+      <div className={topContainerClasses}>
+        {imgSrc && (
+          <Image
+            src={imgSrc}
+            alt={name ? name : ''}
+            lazy={EImageLoadingType.EAGER}
+          />
+        )}
+      </div>
+      {editable && (
+        <div className="absolute bottom-0 right-0">
+          <input
+            style={{ display: 'none' }}
```

Comment:
instead of using inline styles, you could use the tailwind class `hidden`  in className prop to set `display: none `